twisted/python/components.py

```diff
@@ -396,7 +396,7 @@ def __call__(self, oself, *args, **kw):
 class _ProxyDescriptor(object):
     """
     A descriptor which will proxy attribute access, mutation, and
-    deletion to the L{original} attribute of the object it is being accessed
+    deletion to the original attribute of the object it is being accessed
```

Comment:
Maybe we can use L{_ProxyDescriptor.orginalAttribute} of of not maybe I{originalAttribute}